src/Examine.Lucene/Search/LuceneSearchExecutor.cs

```diff
@@ -19,15 +23,17 @@ public class LuceneSearchExecutor
         private readonly ISearchContext _searchContext;
         private readonly Query _luceneQuery;
         private readonly ISet<string> _fieldsToLoad;
+        private readonly IList<IFacetField> _facetFields;
         private int? _maxDoc;
 
-        internal LuceneSearchExecutor(QueryOptions options, Query query, IEnumerable<SortField> sortField, ISearchContext searchContext, ISet<string> fieldsToLoad)
+        internal LuceneSearchExecutor(QueryOptions options, Query query, IEnumerable<SortField> sortField, ISearchContext searchContext, ISet<string> fieldsToLoad, IList<IFacetField> facetFields)
```

Comment:
Ilist is mutable, this should be an immutable collection, like IEnumerable<IFacetField>